5.6/apache/Dockerfile

```diff
@@ -50,5 +50,5 @@ RUN { \
     && a2enmod rewrite
 
 # Install composer
-COPY install-composer /install-composer
+COPY image-files /
```

Comment:
> You won't get less layers than one for all possible files we need to add - which we have with this method.

We don't need the extra `COPY` and `RUN` statement, so it's 2 fewer layers.

The install script is so trivial that there can't be frequent changes. If it ever changes it makes no difference if we only have one file or if we have to go through a hadn full of Dockerfiles. The builds would break anyway so we'd notice for sure, if we forget something.